README.md

```diff
@@ -84,6 +84,25 @@ two properties in the "Common" section
 copy of the `./config/marklogic-connect-standalone.properties` file. This file already has the two properties set to the
   correct values, so you should not need to do anything further with them.
 
+### Dead Letter Configuration
+
+The connector may also be configured to use a dead letter queue (DLQ) when the connector is unable to successfully
```

Comment:
I think we want a different approach here - this is explaining how to setup DLQ, and we don't want to be in the business of explaining how to configure Kafka - the user is either assumed to know how to do that, or is expected to figure that out from Kafka/Confluent docs. At the most, providing the link to https://www.confluent.io/blog/kafka-connect-deep-dive-error-handling-dead-letter-queues/ is enough .

Instead, what we want to tell the user is that the sink connector will now (as of 1.8.0) make use of the DLQ if it's been configured. And we have two events to identify:

1. When an incoming Kafka record cannot be converted into a document to write to MarkLogic
2. When a batch of documents (converted from Kafka records) fails to be written to MarkLogic

For 1, we need to explain that only the one failed record will be sent to the DLQ.

For 2, we need to explain that the entire batch of records will be sent to the DLQ, as the connector is not aware of which document caused the failure.

Reply:
We should also document the headers that we add to a record sent to the DLQ, that would be of interest to the user.